modules/services/kubemonkey.tf

```diff
@@ -0,0 +1,33 @@
+variable "dry_run" { default = true }
```

Comment:
Variables belong in `modules/services/variables`

Reply:
Done